src/app/main-header/main-header.component.html

```diff
@@ -1,7 +1,8 @@
 <nav class="navbar justify-content-between temp-header">
     <a class="navbar-brand">
         <img src="../../assets/images/logo.png" height="30" class="d-inline-block align-top" alt="">
-        <span >{{ editionConfig?.title || 'defaultTitle' | translate }}</span>
+        <span class="ml-2 mr-2">{{ editionConfig?.title || 'defaultTitle' | translate }}</span>
```

Comment:
Does this conflict with edition title PR?

Reply:
Probably. I'll rebase this once the edition title PR has been accepted